src/comit_client.ts

```diff
@@ -1,25 +1,49 @@
 import { BitcoinWallet } from "./bitcoin_wallet";
-import { Cnd, SwapRequest, SwapSubEntity } from "./cnd";
+import { Cnd, SwapRequest, SwapSubEntity } from "./cnd/cnd";
+import { Action, EmbeddedRepresentationSubEntity, Entity } from "./cnd/siren";
 import { EthereumWallet } from "./ethereum_wallet";
-import { Action, EmbeddedRepresentationSubEntity, Entity } from "./siren";
 import { Swap } from "./swap";
 
+/**
+ * The ComitClient class is a one-stop shop interface for interacting with {@link Swap}s of {@link Cnd}.
+ *
+ * It bundles all the necessary dependencies ({@link BitcoinWallet}, {@link EthereumWallet}, {@link Cnd}) to
+ * provide you with instances of {@link Swap}s.
+ */
 export class ComitClient {
   private bitcoinWallet?: BitcoinWallet;
   private ethereumWallet?: EthereumWallet;
 
   constructor(private readonly cnd: Cnd) {}
 
+  /**
+   * Injects a {@link BitcoinWallet} into the ComitClient.
+   *
+   * If you are planning to use this instance to handle swaps involving Bitcoin, you should set this.
+   *
+   * @param bitcoinWallet The wallet that should be used to handle Bitcoin related actions of swaps.
```

Comment:
Are you documention the `BitcoinWallet` interface? I think it'd be worth it.

Reply:
I added a comment to the class, but did not add comments to the functions, because I feel they are quite self explanatory. Do you think we should elaborate on that? 
(Would probably do that in a next round / follow up PR...?)

Reply:
added comment to the wallet interface